src/overlays/actors/ovl_En_Ossan/z_en_ossan.c

```diff
@@ -20,227 +129,2379 @@ const ActorInit En_Ossan_InitVars = {
+
+        if (tmp || tmp2) {
+            func_80094520(globalCtx->state.gfxCtx);
+            gDPSetCombineMode(gfxCtx->overlay.p++, G_CC_MODULATEIA_PRIM, G_CC_MODULATEIA_PRIM);
+
+            gDPLoadTextureBlock(gfxCtx->overlay.p++, &D_0400CF40, G_IM_FMT_IA, G_IM_SIZ_8b, 0x10, 0x18, 0,
+                                G_TX_NOMIRROR | G_TX_WRAP, G_TX_NOMIRROR | G_TX_WRAP, 4, G_TX_NOMASK, G_TX_NOLOD,
+                                G_TX_NOLOD);
+
+            if (tmp) {
+                func_80AC77CC(globalCtx, this->unk_26C, this->unk_270, this->unk_274, this->unk_278, this->unk_27C,
+                              this->unk_280, this->unk_284, 0, 0, -1.0f, 1.0f);
+            }
+            if (tmp2) {
+                func_80AC77CC(globalCtx, this->unk_2A4, this->unk_2A8, this->unk_2AC, this->unk_2B0, this->unk_2B4,
+                              this->unk_2B8, this->unk_2BC, 0, 0, 1.0f, 1.0f);
+            }
+
+            gDPLoadTextureBlock(gfxCtx->overlay.p++, &D_0400CE40, G_IM_FMT_IA, G_IM_SIZ_8b, 0x10, 0x10, 0,
+                                G_TX_NOMIRROR | G_TX_WRAP, G_TX_NOMIRROR | G_TX_WRAP, 4, G_TX_NOMASK, G_TX_NOLOD,
+                                G_TX_NOLOD);
+
+            if (tmp) {
+                func_80AC77CC(globalCtx, this->unk_254, this->unk_258, this->unk_25C, this->unk_260, this->unk_264,
+                              this->unk_268, this->unk_284, 0, 0, -1.0f, 1.0f);
+            }
+            if (tmp2) {
+                func_80AC77CC(globalCtx, this->unk_28C, this->unk_290, this->unk_294, this->unk_298, this->unk_29C,
+                              this->unk_2A0, this->unk_2BC, 0, 0, 1.0f, 1.0f);
+            }
+        }
+        Graph_CloseDisps(dispRefs, globalCtx->state.gfxCtx, "../z_en_oB1.c", 4300);
+    }
+}
+
+void func_80AC7DAC(Actor* thisx, GlobalContext* globalCtx) {
+    EnOssan* this = THIS;
+    s32 pad;
+
+    {
+        GraphicsContext* gfxCtx = globalCtx->state.gfxCtx;
+        Gfx* dispRefs[4];
+
+        Graph_OpenDisps(dispRefs, globalCtx->state.gfxCtx, "../z_en_oB1.c", 4320);
+        func_80093D18(globalCtx->state.gfxCtx);
+
+        gSPSegment(gfxCtx->polyOpa.p++, 0x08, SEGMENTED_TO_VIRTUAL(D_80AC8EA0[this->unk_1F2]));
+
+        SkelAnime_DrawSV(globalCtx, this->skelAnime.skeleton, this->skelAnime.limbDrawTbl, this->skelAnime.dListCount,
+                         func_80AC74F4, NULL, &this->actor);
+        func_80AC7528(globalCtx, this, this->unk_230, this->unk_234, this->unk_238, this->unk_251);
+        func_80AC79C8(globalCtx, this);
+        Graph_CloseDisps(dispRefs, globalCtx->state.gfxCtx, "../z_en_oB1.c", 4340);
+    }
+}
+
+#ifdef NON_MATCHING
+// Loading gSegments
+s32 func_80AC7ED0(GlobalContext* globalCtx, s32 limbIndex, Gfx** dList, Vec3f* pos, Vec3s* rot, Actor* thisx) {
+    EnOssan* this = THIS;
+    s32 pad;
+
+    {
+        GraphicsContext* gfxCtx = globalCtx->state.gfxCtx;
+        Gfx* dispRefs[4];
+
```

Comment:
only use the macro blocks if you have to, this creates more work for the person who implements the macro